README.rst

```diff
@@ -87,83 +87,88 @@ Community
+   consistent.
+
+We are excited to welcome your contributions to PypeIt!  We acknowledge 
+contributions take many forms, including but not limited to participating in
+discussions in our Users Slack Workspace; reporting issues to our GitHub
+repository; submitting pull requests with small bug fixes, documentation
+improvements, or large feature improvements; and participating in project
+maintenance and governance.  All contributors are expected to follow our `Code
+of Conduct <https://pypeit.readthedocs.io/en/stable/codeconduct.html>`__.
+
+For direct contributions to the code, please see our `Development Guidelines
+<https://pypeit.readthedocs.io/en/stable/dev/development.html>`__.  As mentioned
+therein, communication between developers is key to ensuring efforts are
+coordinated.  Before beginning any development activities, we would appreciate
+communicating your intentions to the core development team, e.g., via the PypeIt
+Users Slack Workspace.
+
+For information regarding our governance structure and policies, please see the
+`PypeIt Governance <https://pypeit.readthedocs.io/en/stable/governance.html>`__
+documentation.
+
+For a list of current contributors and project roles, please see our `PypeIt
+Team <https://pypeit.readthedocs.io/en/stable/team.html>`__ listing.
+
+----
+
 Citation
 ++++++++
 
 If you use PypeIt in your research, please cite the following publications
 (we provide the relevant `BibTeX entries
-<https://pypeit.readthedocs.io/en/release/index.html#pypeit-bibtex-entries>`__
+<https://pypeit.readthedocs.io/en/stable/index.html#pypeit-bibtex-entries>`__
 for your convenience):
 
  - Prochaska et al. (2020, JOSS): `arXiv <https://ui.adsabs.harvard.edu/abs/2020arXiv200506505P/abstract>`__, `JOSS <https://joss.theoj.org/papers/10.21105/joss.02308>`__
  - Prochaska et al. (2020, Zenodo): `Zenodo <https://ui.adsabs.harvard.edu/abs/2020zndo...3743493P/abstract>`__
 
-If there is no place to include the relevant citations in the text of
-the publication, please include the following acknowledgement
-(provided in latex and using the provided `BibTeX entries
-<https://pypeit.readthedocs.io/en/release/index.html#pypeit-bibtex-entries>`__):
+You are also encouraged to note the specific version of the code you have used
+(e.g., 1.17.3).  If there is no place to include the relevant citations in the
+text of the publication, please include the following acknowledgement (provided
+in latex and using the provided `BibTeX entries
+<https://pypeit.readthedocs.io/en/stable/index.html#pypeit-bibtex-entries>`__):
 
 .. code-block:: latex
 
-    This research made use of \ttfamily{PypeIt},\footnote{\url{https://pypeit.readthedocs.io/en/latest/}}
-    a Python package for semi-automated reduction of astronomical slit-based spectroscopy
+    This research made use of \ttfamily{PypeIt} version
+    1.17.3,\footnote{\url{https://pypeit.readthedocs.io/en/stable/}} a Python
```

Comment:
might be worth looking for a way to ideally automate generating the version number here or at least add it to the release checklist.